docs/topics/modules/running_your_project_in_the_che_workspace.adoc

```diff
@@ -26,25 +25,16 @@ See <<create_run_command_macro>> for instructions on setting up a run command.
 +
 . A *run* terminal appears at the bottom pane of the Che workspace. When the `mvn{nbsp}build` command finishes executing, the *run* view displays the *Succeeded in Deploying verticle* message:
 +
-image::{context}_deployed_verticle.png[Successfully deployed verticle]
+----
+[INFO] INFO: Succeeded in deploying verticle
```

Comment:
Do we need this in addition? The step tells the user about the displayed message. 
Deploying>deploying.

Reply:
Slightly changing text. I think the actual message appended to the end is useful in terms of verification so removing that specific text from the line before it.